addons/account_bank_statement_import/migrations/12.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,5 @@
+---Fields in module 'account_bank_statement_import'---
```

Comment:
Files swapped. This one is the original one.

Reply:
Right. Amended! 😂